src/Language/PureScript/TypeChecker/Deriving.hs

```diff
@@ -1,20 +1,25 @@
+{-# LANGUAGE GADTs #-}
```

Comment:
Is this something you were using initially and later removed? I don't think I see a usage of this in this file.

Reply:
This is used in the definition of `TraversalOps`, which has an existential quantifier.

Reply:
Ah... ok. I'll mark this as resolved.